### PR TITLE
Obg 421/feature ng tests consent ui

### PR DESCRIPTION
**Pull request by vladimir-nechai**
- fixed tests
- ng test to travis ci

### CODE REVIEW

File: consent-ui/src/app/ais/entry-page/initiation/common/initial-consent/consent-account-access-selection.component.spec.ts [hunk 1]
```diff
@@ -1,21 +1,52 @@
...
-      declarations: [ ConsentAccountAccessSelectionComponent ]
-    })
-    .compileComponents();
+      declarations: [ConsentAccountAccessSelectionComponent],
+      imports: [ReactiveFormsModule, RouterTestingModule],
+      providers: [
+        {
+          provide: ActivatedRoute,
+          useValue: {
+            parent: { parent: { params: of({ authId: 'AUTH-ID' }) } },
```
Comment (valb3r) [inline consent-ui/src/app/ais/entry-page/initiation/common/initial-consent/consent-account-access-selection.component.spec.ts]: Suggest to replace multiple occurences of 'AUTH-ID' with some constant

File: consent-ui/src/app/ais/entry-page/initiation/consent-initiate/consent-initiate.component.spec.ts [hunk 2]
```diff
@@ -1,16 +1,26 @@
...
     TestBed.configureTestingModule({
-      declarations: [ ConsentInitiateComponent ]
-    })
-    .compileComponents();
+      declarations: [ConsentInitiateComponent],
+      imports: [ReactiveFormsModule, HttpClientTestingModule, RouterTestingModule],
+      providers: [
+        {
+          provide: ActivatedRoute,
+          useValue: { snapshot: { params: { authId: 'AUTH-ID' }, queryParams: { redirectCode: 'REDIRECT-CODE' } } }
```
Comment (valb3r) [inline consent-ui/src/app/ais/entry-page/initiation/consent-initiate/consent-initiate.component.spec.ts]: Suggest to replace multiple occurrences of 'REDIRECT-CODE' with some constant

File: .travis.yml [hunk 3]
```diff
@@ -13,6 +13,7 @@ script:
   # tests are not done with mvn test - dont know why
   - ./scripts/build_mvn.sh
   - docker run -it --rm -v "$PWD":/opt/app-root/src -w /opt/app-root/src -u "$(id -u "${USER}"):$(id -g "${USER}")" trion/ng-cli make fintech-ui
+  - docker run -it --rm -v "$PWD":/opt/app-root/src -w /opt/app-root/src -u "$(id -u "${USER}"):$(id -g "${USER}")" trion/ng-cli-karma make consent-ui-test
```
Comment (valb3r) [inline .travis.yml]: Technically we may end in the situation that we are testing with one toolset and compiling with other as containers are different... Maybe we can build and test in one container?

File: consent-ui/src/app/ais/entry-page/initiation/common/initial-consent/consent-account-access-selection.component.spec.ts (hunk 1 above)
Comment (vladimir-nechai) [reply consent-ui/src/app/ais/entry-page/initiation/common/initial-consent/consent-account-access-selection.component.spec.ts]: done

File: consent-ui/src/app/ais/entry-page/initiation/consent-initiate/consent-initiate.component.spec.ts (hunk 2 above)
Comment (vladimir-nechai) [reply consent-ui/src/app/ais/entry-page/initiation/consent-initiate/consent-initiate.component.spec.ts]: done

File: .travis.yml (hunk 3 above)
Comment (vladimir-nechai) [reply .travis.yml]: totally makes sense. Fixed

**Pull request merged**